docs/usage/faq.md

```diff
@@ -39,6 +39,26 @@ The Renovate team will only create bugfixes for an older version if the hosted a
 If you're using the hosted app, you don't need to do anything, as the Renovate maintainers update the hosted app regularly.
 If you're self hosting Renovate, use the latest release if possible.
 
+## Renovate core features not supported on all platforms
+
+| Feature                                           | Platforms which lack feature                      | See Renovate issue(s) |
+| ------------------------------------------------- | ------------------------------------------------- | --------------------- |
+| Dependency Dashboard                              | BitBucket, BitBucket Server, Azure                | #9592                 |
+| Hosted app                                        | GitLab, BitBucket, BitBucket Server, Azure, Gitea |                       |
+| Cloning repo over SSH selfhosted                  | GitLab                                            | #5406                 |
+| Select which status checks to apply for automerge | Azure DevOps                                      | #4822                 |
+| Configurable merge strategy and message           | BitBucket, BitBucket Server                       | #7184                 |
+
+## Major platform features not suported by Renovate
```

Comment:
The idea here is to list some major "nice to have/unique selling point" features that are platform specific, that we don't support, so things like the merge-trains on GitLab, Jira issues...

Reply:
Adding the configurable required status checks feature here?